yopt/src/main/scala/pravda/yopt/CmdDecoder.scala

```diff
@@ -93,4 +232,25 @@ object CmdDecoder {
 
     override val optInfo: Option[String] = None
   }
+  
+  private def sequence[A, E](list: Traversable[Either[E, A]]): Either[E, Traversable[A]] = {
```

Comment:
You can use `sequence` from `cats`, it's a core dependency.

Reply:
Yes, will be fixed. But my Intellj Idea marks such code with red color even the "-Ypartial-unification" scalac option is presented.

Reply:
Yeah, I know :) But it's not a reason to write boilerplate.